config/lava/chromeos/chromeos-tast-template.jinja2

```diff
@@ -5,44 +5,22 @@
 {% endblock %}
 {% block actions %}
 actions:
-  - deploy:
+- deploy:
```

Comment:
I think the indentation was fine, what's the reason for this change?

Reply:
Jenkins job was failing.
Here is failed example: https://bot.staging.kernelci.org/job/chromeos/job/test-runner/5182/console
I generated yaml manually also and confirmed that octopus was failing.